src/overlays/actors/ovl_player_actor/z_player.c

```diff
@@ -1130,65 +1130,117 @@ static LinkAnimationHeader* D_80853D7C[][2] = {
     { &gPlayerAnim_link_wait_itemD2_20f, &gPlayerAnim_link_wait_itemD2_20f }
 };
 
-static AnimSfxEntry D_80853DEC[] = {
+static AnimSfxEntry sIdleSneezeSfxEntry[] = {
```

Comment:
Given that AnimSfxEntry represents a single element in a list of elements, shouldn't these be name like `sIdleSneezeAnimSfx` instead?

Reply:
Done.